pkg/registry/servicecatalog/rest/storage_servicecatalog.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rest
+
+import (
+	"k8s.io/kubernetes/pkg/api/rest"
+	"k8s.io/kubernetes/pkg/genericapiserver"
+	"k8s.io/kubernetes/pkg/registry"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
+	servicecatalogv1alpha1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	"github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/binding"
+	"github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/broker"
+	"github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/instance"
+	"github.com/kubernetes-incubator/service-catalog/pkg/registry/servicecatalog/serviceclass"
+)
+
+// StorageProvider provides a factory method to create a new APIGroupInfo for
+// the servicecatalog API group.
+type StorageProvider struct{}
+
+// NewRESTStorage is a factory method to make a new APIGroupInfo for the
```

Comment:
My preference is `NewStorage` so its usage reads `rest.NewStorage`

Let's leave as-is and come back to it